src/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfoCommon.cs

```diff
@@ -160,23 +160,25 @@ public override IEnumerable CreateDerivedEnumerableInstance(JsonPropertyInfo col
 
             // TODO: Use reflection to support types implementing Stack or Queue.
```

Comment:
What does this comment refer to?  Is there an issue tracking this? (I realize it's separate from your change.)

Reply:
@layomia?

Reply:
This refers to providing deserialization support for more implementing types (types that implement an enumerable that is natively supported by the serializer). I'll create an issue for this shortly and add a link to it in the corresponding comment in the master branch. We might not provide the support but I think it should be considered.

Reply:
Is there an issue to track a final decision on that?

Reply:
https://github.com/dotnet/corefx/issues/40479.